module/applications/dice/damage-configuration-dialog.mjs

```diff
@@ -81,8 +81,10 @@ export default class DamageRollConfigurationDialog extends RollConfigurationDial
 
   /** @override */
   _finalizeRolls(action) {
+    const isCritical = action === "critical";
+    this.config.isCritical = isCritical;
     return this.rolls.map(roll => {
-      roll.options.isCritical = action === "critical";
+      roll.options.isCritical = isCritical;
```

Comment:
We can save this extra assignment by just applying `this.config.isCritical` to the roll options, since it will always be the same as `isCritical`.

```suggestion
      roll.options.isCritical = this.config.isCritical;
```